alf/__init__.py

```diff
@@ -15,6 +15,18 @@
 from .config_util import *
 from .tensor_specs import *
 
+# NOTE: Run ``import torch.fx`` before importing ``alf.module`` is a workaround
+# to tame ``torch.fx``. When ``torch.fx`` is first imported and evaluated,
+# ``Module.__getattr__`` is accessed. This access will raise an error if it is
+# already deleted by ``alf.module``.
+#
+# Newer version of torchvision relies on ``torch.fx``, and there can be more use
+# of ``torch.fx`` in the future.
+try:
+    import torch.fx
+except ModuleNotFoundError:
+    pass
```

Comment:
HOW  about put these in module.py

Reply:
Definitely better and I should have put it there. Updated.